setup_8.x

```diff
@@ -0,0 +1,295 @@
+#!/bin/bash
+
+# Discussion, issues and change requests at:
+#   https://github.com/nodesource/distributions
```

Comment:
Wow, there really isn't a package for node 8 on Ubuntu 16?  This is really the best option?  Maybe I was wrong about it being easier to put Node on a Parity image vs Parity on a node image?

Anyway, it is fine... just lamenting that everything is terrible.

Reply:
Now that we're talking about, perhaps I should just lift their steps from this? I think what I'll do: Make our own parity+node8 using this stuff:

https://github.com/nodejs/docker-node/blob/master/Dockerfile-stretch.template

Then build the rest on it? Since that is more straight-forward apt stuff that feels cleaner